src/sentry/conf/server.py

```diff
@@ -914,6 +914,7 @@ def create_partitioned_queues(name):
         "SERVERS": [{"url": "https://sentry.io/"}],
         "PARSER_WHITELIST": ["rest_framework.parsers.JSONParser"],
         "APPEND_PATHS": get_old_json_paths(OLD_OPENAPI_JSON_PATH),
+        "SORT_OPERATION_PARAMETERS": False,
```

Comment:
If anyone is curious about this, this makes it so that our parameters are listed in the order that we define them in, instead of alphabetically.